blocks/quote/quote.js

```diff
@@ -0,0 +1,11 @@
+export default function decorate(block) {
```

Comment:
Modify the `quote` block so that the textContent and author are dynamic (coming from the source)

Reply:
Use semantic HTML for the quote